webpack/JobWizard/steps/AdvancedFields/DescriptionField.js

```diff
@@ -43,9 +56,20 @@ export const DescriptionField = ({ inputs, value, setValue }) => {
       }
     >
       {isPreview ? (
-        <TextInput id="description-preview" value={generatedDesc} isDisabled />
+        <Tooltip content={generatedDesc}>
+          <div>
+            {/* div wrapper so the tooltip will be shown in chrome */}
```

Comment:
is there an open issue for it in PF4?

Reply:
https://github.com/patternfly/patternfly-react/issues/1894